data/json/monsters/monsters.json

```diff
@@ -3107,6 +3107,54 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_wretched",
+    "looks_like": "mon_zombie_rot",
+    "type": "MONSTER",
+    "name": "wretched puker",
+    "description": "A degenerate corpse, shambling as it walks. Several junks and waste materials have unified into its skin and wounds around it reeks in black goo.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "//": "Higher weight from attached junks.",
+    "weight": 84500,
+    "hp": 55,
+    "speed": 65,
+    "material": [ "flesh" ],
+    "symbol": "Z",
```

Comment:
```suggestion
    "symbol": "Z",
    "looks_like": "mon_zombie_acidic",
```

Reply:
Similar sprite to its upgraded counterpart... But anyways, "someone" might make its own sprite 😄

Reply:
Oh, looking at the changes, I didn't realize you already had a looks like for it. There was no need to change it.
Sorry for missing that.

Reply:
It's not a trouble. I've put it in and let it be there now :wink: